src/components/Tooltip/index.js

```diff
@@ -149,7 +145,7 @@ class Tooltip extends PureComponent {
                         onHoverIn={this.showTooltip}
                         onHoverOut={this.hideTooltip}
                     >
-                        {React.Children.only(this.props.children)}
```

Comment:
Removed both `isValidElement` and `React.Children.only` from Tooltip because Hoverable already handle the `React.Children.only` and `React.cloneElement` won't work on invalid element.